services/httpd/handler.go

```diff
@@ -202,6 +208,22 @@ func NewHandler(c Config) *Handler {
 			"delete",
 			"POST", "/api/v2/delete", false, true, h.serveDeleteV2,
 		},
+		Route{
+			"buckets",
+			"POST", "/api/v2/buckets", false, true, h.serveBucketsV2,
```

Comment:
Maybe call this servePostBucketsV2 or something to emphasize we're going to create a bucket